helmfile/charts/grafana-ops/dashboards/elasticsearch-dashboard.json

```diff
@@ -6736,8 +6736,7 @@
   "schemaVersion": 30,
   "style": "dark",
   "tags": [
-    "elasticsearch",
-    "App"
+    "opensearch"
```

Comment:
Could you rename this file as well? from `elasticsearch-dashboard` to `opensearch-dashboard`